userdocs/diagnostics/multiple-class-inheritance.md

```diff
@@ -0,0 +1,37 @@
+# Multiple Inheritance of Classes
+
+In some programming languages, a class can have multiple superclasses. This feature is called multiple inheritance and is unsupported in Swift. However, using protocols, one can achieve a similar effect as multiple inheritance.
```

Comment:
```suggestion
In some programming languages, a class can inherit the interface of multiple superclasses. Known as multiple inheritance, this feature can add significant complexity to the language and is unsupported in Swift. Instead, Swift allows composition of interfaces using protocols.
```

I think it's important to state (however briefly or generally) a rationale why something isn't supported. Readers are naturally inclined to seek out reasons. Without something there, the text raises ambiguity as to whether this lack of support is temporary and unintentional, or even if the reader correctly surmises that it is intentional, leaves them guessing as to why.